todo/src/actions/action.js

```diff
@@ -0,0 +1,27 @@
+export const ADD_TODO = "ADD_TODO";
+export const DELETE_TODO = "DELETE_TODO";
+export const COMPLETE_TODO = "COMPLETE_TODO";
+export let idGenerator = 3;
+
+export const addTodo = text => {
```

Comment:
While this works just fine, action creator usually only have a `type` and  optional `payload` keys. So everything else should be put into the payload.
```
  return {
    type: ADD_TODO,
    payload: {
      id: idGenerator++,
      completed: false,
      text
    }
  }
```